build.cmd

```diff
@@ -19,16 +19,18 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 rem copy nuget locally
-if not exist .nuget\NuGet.exe (
-  if not exist .nuget md .nuget
-  copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
-)
+if not exist .nuget md .nuget
+copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul

```

Comment:
Ah, yes, I wanted to add a comment about this but forgot.

> why not `/Y` instead of `> nul`?

Well, you're the one who [wrote it](https://github.com/thomaslevesque/FakeItEasy/blob/cb2db7b0700508e542fd09ad768d55345b7f9681/build.cmd#L24), so I thought you'd know :wink:
I just left it as it was. I can change it if it makes sense.

> (my preference) why do we bother copying to the local directory at all? We're always making sure there's something in the cache directory. We could just run from there.

I wondered about this too. I'm not sure I see the point of copying either. The only reason I can see is that this way, the csx script doesn't need to know about the cache location. If we want to run directly from the cache dir, we need to somehow tell the script where to find NuGet (maybe via an environment variable). This will make it harder to run the csx manually, without using the cmd, which is probably not a big issue, since we're unlikely to want to do that.